api/libproject/wboptionsparser.rb

```diff
@@ -46,7 +46,8 @@ def validate()
     @validators.each do |fn|
       begin
         fn.call(@opts)
-      rescue ArgumentError
+      rescue ArgumentError => e
+        puts e.message unless e.message.nil?
```

Comment:
nit: should this go to STDERR as well? (along with the help text on the next line)